shared/ui/src/commonMain/kotlin/com/adammcneilly/pocketleague/shared/ui/FeedContent.kt

```diff
@@ -0,0 +1,36 @@
+package com.adammcneilly.pocketleague.shared.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+/**
+ * The main list of events and matches to show within the feed screen
+ * that is the landing page when opening the app.
+ */
+@Composable
+fun FeedContent(
+    modifier: Modifier = Modifier,
+) {
+    LazyColumn(
+        modifier = modifier,
+        contentPadding = PaddingValues(16.dp),
```

Comment:
Let's move this into some sort of spaces collection and extend material theme: https://developer.android.com/jetpack/compose/designsystems/custom#extending-material